### ISSUE TITLE
Input event does not fire with non-latin characters

### ISSUE THREAD

**Issue by ArmorDarks** [ionic-team/capacitor#757]
## Version 

1.0.0-beta.4

## Platform

Android

Tested and recreated on Android 8.0 and Android 6.0

Recreatable only in built Android app, not PWA

## Issue

When inputting non-latin characters into `input` field, `input` event doesn't fire.

This code recreates the issue:

```html
<input id='TEST_ME' />
<button id='BUTTON'>check</button>

<script>
  TEST_ME.addEventListener('input', (e) => {
    console.log('value target', e.target.value, typeof e.target.value)
    console.log('value elemen', TEST_ME.value, typeof TEST_ME.value)
  })

  BUTTON.addEventListener('click', () => {
    console.log('checked value', TEST_ME.value, typeof TEST_ME.value)
    console.log('checked value', TEST_ME)
  })
</script>
```

If `test` would be filled into the input, it will log content. However, if something with Cyrillic characters, like `абц`, would be entered, there won't be anything logged to the console.

However, upon clicking `check` it would be shown, that `TEST_ME` value actually there. It's just that `input` event does not fire.

Note, that issue is quirky. For instance, if into input would be entered `абц` and then typed any number, or latin character, `input` event firing will suddenly start to work.

Also, deleting characters seems to fire event too, even if those characters are Cyrillic.

Entering characters through WebView inspector _will not trigger issue_, same as copying them withing inspector or on the device itself.

To recreate the issue, characters like `а` `б` `ц` should be exactly typed in one by one with any (in my case default) keyboard on the Android device (I guess simulator would work too).

I've tested with Ukrainian and Russian languages — both affected by this issue.

Unfortunately, this issue is quite sever, since it makes most apps unusable in Ukraine and Russia, especially SPA-ones, like Vue, which relies on input events triggering for writing values into the state.

**Comment by jcesarmobile** [comment]
This was fixed in beta 6, in the PR you linked (#717)

**Comment by ArmorDarks** [comment]
Oh God, beta 6 indeed fixed the issue, tested just right now. Thanks, you've saved my day!

Tracking changes without changelog sometime is tricky...

Do you accept donations anywhere?
